Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -314,6 +337,8 @@ extension ActivityStreamPanel: UICollectionViewDelegateFlowLayout {
             return CGSize(width: cellSize.width, height: estimatedLayout.size.height)
         case .pocket:
             return cellSize
+        case .libraryShortcuts:
+            return CGSize(width: min(ASPanelUX.LibraryShortcutsMaxWidth, cellSize.width), height: (cellSize.width/4)+10)
```

Comment:
nit: spacing on operators. Also comment on what div 4 plus 10 is (or perhaps there is a constant that the 4 is related to?)